application/mounts.py

```diff
@@ -42,11 +42,21 @@ def _add_mount(self, monitor, mount):
 		mount_uri = mount.get_root().get_uri()
 		volume = mount.get_volume()
 
+		# check if mount has been added as volume already
+		mount_already_in_volume_list = False
```

Comment:
Thanks for your consideration. I would merge it eventually just am kind of short on free time.